lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is a group of items in which each item has an attribute or value stored along with it.
```

Comment:
Your definition is: "group of items in which each item has an attribute or value stored along with it" 

So, thinking of a Hash visually, consider the following data:

'item' | 'attribute or value stored along with it'
------ | -----------------------
"winner" | "Mindy"
"runner-up" | "Daniel"
"bronze" | "Sumeet"

Notice that I've used your definition's components as the column headers in the table.

I have two questions, referencing the above information:

1. What Ruby code would you write to express that information as a Hash?
2. What are the correct terms for the column headers?